app/services/umrdr/deposits_okay_service.rb

```diff
@@ -0,0 +1,59 @@
+module Umrdr
+
+  require 'rubygems'
+  require 'net/ldap'
+
+  ENV['LDAPTLS_CACERT'] = '../../../config/incommon_server_ca_cert.crt';
+
+  class DepositsOkayService
```

Comment:
Aaron suggested in https://github.com/mlibrary/umrdr/issues/263 that we would want to build in a check for membership in an ldap group controlled by the library research team. So, I was keeping the name a bit more general. Of course, that could be a separate services.